.github/workflows/main-cmake.yml

```diff
@@ -42,6 +42,7 @@ jobs:
           run: echo '${{ toJSON(matrix) }}'
 
         - run: echo "CMAKE_PREFIX_PATH=${env.CONDA_PREFIX}/Library" >> $GITHUB_ENV
+        - run: echo "PATH=${CONDA_PREFIX}/Library/lib:${PATH}" >> $GITHUB_ENV
```

Comment:
Path separator on windows is `;`. Also Github actions doe not use the `PATH` in `GITHUB_ENV`. It usese `GITHUB_PATH`

Reply:
Since I'm working in an MSYS2 shell, the guidance I'm finding seems to be that I should use `:` instead of `;`.

Reply:
However, clearly something is wrong, so I'll take a look. My priority is to get something that *works* so that I can move on with the release process, replacing Appveyor, and refine it afterwards. Premature optimization, and all that.